docs/source/install.rst

```diff
@@ -54,12 +53,19 @@ The ROSCO binary file can be copied to your desired folder using:
 
     cp $CONDA_PREFIX/lib/libdiscon.* .
 
+on linux or:
```

Comment:
The OpenFAST documentation has this problem where we have many `if mac/linux... otherwise for windows ...`. As a possible solution, I'm planning to try adding embedded tabs for difference OS's; something like https://getbootstrap.com/2.3.2/components.html#navs. It might be relevant here, as well.

Reply:
A really good suggestion. If it's okay, I'd like to get this merged in and then perhaps another restructure later if needed. Already having trouble keeping up with `develop`.